web-api/src/mind/config.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+use xayn_ai_coi::CoiConfig;
+use xayn_test_utils::error::Panic;
```

Comment:
I assume we use panic to get rid of the `unwraps`?

Reply:
exactly, as noted in the pr description